define/fsd.md

```diff
@@ -357,6 +422,19 @@ The enumerated values are comma-delimited alphanumeric names surrounded by brace
 
 Enumerated values are always transmitted as strings, not integers.
 
+
+#### Validation
```

Comment:
This second Validation section seems redundant with the first.

Reply:
Indeed. I began to add it there but decided against it and forgot to remove it.